packages/integration-sdk-core/src/types/step.ts

```diff
@@ -48,10 +49,6 @@ export type ExecutionHandlerFunction<T extends StepExecutionContext> = (
   context: T,
 ) => Promise<void> | void;
 
-export type StepExecutionHandlerFunction<
-  TConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
-> = ExecutionHandlerFunction<IntegrationStepExecutionContext<TConfig>>;
-
```

Comment:
There are no references to this type: https://cs.github.com/?scope=org%3AJupiterOne&scopeName=JupiterOne&q=StepExecutionHandlerFunction